### ISSUE TITLE
Class distribution (all games)

### ISSUE THREAD

**Issue by avandenberghe** [avatharbe/bbguild#279]
Remaining scope after #366 (Guild statistics portal module, shipped 2026-09-12):

- Graph/chart visualization of the distributions (originally specced against the Flot library; a currently-maintained charting lib should be used instead since Flot is long dead)
- Optional build list

The numbers/percentage + class distribution part of this issue is done — see #366 (game-agnostic `portal\modules\statistics` module: class/race/level distribution, rank/active-member counts, recent joins/departures).

---
Original request:
add a guild statistics module

- by percentage and numbers
- with graph library (flot).
- optionally add build list
- class distribution
- etc

**Comment by avandenberghe** [comment]
Closing as superseded by #366 (Guild statistics portal module, shipped 2026-09-12) — bbguild's own roadmap doc already documented #366 as subsuming this one. Note the gap: #366 covers class/race/level/rank distribution + recent joins/departures as simple lists, but not this issue's Flot graph-library charting or the optional build-list feature. If those are still wanted, worth filing as a new, narrower issue rather than reopening this one.

**Comment by avandenberghe** [comment]
Reopening — closure was premature. #366 only delivered the numbers/percentage side of this (class/race/level/rank distribution as lists); the chart/graph visualization (Flot) and the optional build-list feature this issue asked for were never built. Narrowing scope to just those two remaining items; the distribution-list part is done via #366.
